client/my-sites/checkout/cart/cart-messages.jsx

```diff
@@ -83,6 +99,11 @@ function getPrettyErrorMessages( messages, { translate, selectedSiteSlug } ) {
 					message: getBlockedPurchaseErrorMessage( { translate, selectedSiteSlug } ),
 				} );
 
+			case 'invalid-product-multisite':
+				return Object.assign( error, {
+					message: getInvalidMultisitePurchaseErrorMessage( { translate, ...error } ),
```

Comment:
Minor, but would passing `message` explicitly be more clear? Otherwise we're potentially passing in a bunch of arguments that aren't used.

```suggestion
					message: getInvalidMultisitePurchaseErrorMessage( { translate, message: error.message } ),
```